scripts/ec2cleanup.py

```diff
@@ -189,6 +215,10 @@ def ec2cleanup(exclude_volumes, exclude_eips, exclude_elbs, exclude_enis, exclud
                       excluded_stacks=exclude_stacks,
                       stack_template=stack_template,
                       output=output)
+        logger.info("Deleting snapshots...")
+        delete_snapshots(provider_mgmt=provider_mgmt,
```

Comment:
Q: shouldn't your methods pass something to sys.exit ? 
what's the reason for having sys.exit if nothing is passed there ?